WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
```

Comment:
It prevents Tsakonian **_GREEKS_** and Thracian _**GREEKS**_ (etc.) from utilizing the title. I simply fixed a mistake.